doc/UsersGuide/source/CustomizingTheWorkflow/PhysicsSchemes.rst

```diff
@@ -0,0 +1,132 @@
+
+Shallow convection is similar to deep convection, except that the base mass flux is averaged based on the updraft velocity in a cloud layer. Only convection updrafts are considered in the shallow scheme. HAFS tests show that HAFS is sensitive to entrainment and detrainment rates. HFSA uses an entrainment rate larger than that in NCEP GFS.
+
+.. _MicrophysicsSchemes:
+
+====================
+Microphysics Schemes
+====================
+
+**Effects of Vapor-Liquid-Ice Phase Changes:**
+
+HAFS uses GFDL microphysics and Thompson microphysics.
+
+**GFDL:**
+
+The GFDL microphysics is a single-moment scheme that includes QC, QI, QR, QS, and QG. It is based on the Lin-Lord-Krueger cloud microphysics scheme.
+
+**Thompson:**
+
+The Thompson microphysics scheme includes QC, QI, QR, QS, QG, and the number concentration of cloud ice, cloud water, and rain. It also considers the impact of aerosols. HAFS performs well with the Thompson scheme for TCs in the NATL basin and with GFDL for TCs in the EAPC basin.
+
+.. _RadiationSchemes:
+
+=================
+Radiation Schemes
+=================
+
+Radiation schemes are used for heating and cooling due to short- and long-wave radiation.
+
+HAFS uses the RRTMG radiation scheme. For computational efficiency, the correlated K-method is used: The shortwave algorithm includes 112 g-points (quadrature points) in 14 bands, while the longwave algorithm includes 140 unevenly distributed g-points in 16 broad spectral bands. Aerosol optical properties, cloud liquid water and ice paths, and effective radius are used to represent the radiative effects of aerosols and clouds in the calculation. The effects of sub-grid scale clouds are treated by a Monte-Carlo Independent Column Approximation (McICA) method, with a decorrelation length overlap assumption for multi-layered clouds.
+
+.. _GravityWaveDrag:
+
+=================
+Gravity Wave Drag
+=================
+
+Impact of sub-grid scale perturbations excited by orography and convection.
+
+**HAFS uses the unified GWD scheme:**
+
+Mesoscale orographic gravity wave drag, low-level flow blocking by subgrid-scale orography, effects of gravity waves produced by horizontal terrain variations, and non-topographic gravity wave drag (GWD) from convection and frontal instability are considered in the model.
+
+.. _SuiteComparison:
+
+================
+Suite Comparison
+================
+
+The following table compares the different configurations for Suite 1 (HFSAv2) and Suite 2 (HFSBv2):
+
+.. list-table:: Suite Comparison
+   :header-rows: 1
+
+   * - Component
+     - Suite 1 (HFSAv2)
+     - Suite 2 (HFSBv2)
+   * - Land/ocean Surface
+     - NOAH LSM VIIRS veg type, MOM6
+     - NOAH LSM VIIRS veg type, HYCOM
+   * - Surface Layer
+     - GFS, HWRF TC-specific sea surface roughnesses
+     - GFS, HWRF TC-specific sea surface roughnesses
+   * - Boundary Layer
+     - Sa-TKE-EDMF+shear: sfc_rlm=1, tc_pbl=0, elmx/rlmx=250 (nest)
+     - Sa-TKE-EDMF+shear: sfc_rlm=0, tc_pbl
```

Comment:
@jdkublnick I think the end of this table is missing. When you have a chance, could you push whatever changes you have locally and bold anything that is different between the HFSA and HFSB configs?